tools/cldr-code/src/test/java/org/unicode/cldr/unittest/TestPathHeader.java

```diff
@@ -1677,4 +1679,70 @@ public void TestUnitOrder() {
             }
         }
     }
+
+    public void testPageSize() {
+        final long minError = 600; // above this, emit error
+        final long minLog = 500; // otherwise above this, emit warning
+        Factory factory = CLDRConfig.getInstance().getCommonAndSeedAndMainAndAnnotationsFactory();
+        // "en", "cs", "ar", "pl"
+        List<String> locales =
+                StandardCodes.make()
+                        .getLocaleCoverageLocales(Organization.cldr, ImmutableSet.of(Level.MODERN))
+                        .stream()
+                        .filter(x -> CLDRLocale.getInstance(x).getCountry().isEmpty())
+                        .collect(Collectors.toUnmodifiableList());
+        List<Counter<PageId>> counters = new ArrayList<>();
+        for (String locale : locales) {
+            CLDRFile cldrFile = factory.make(locale, false);
+            PathHeader.Factory phf = PathHeader.getFactory();
+            Counter<PageId> c = new Counter<>();
+            counters.add(c);
+            for (String path : cldrFile) {
+                PathHeader ph = phf.fromPath(path);
+                c.add(ph.getPageId(), 1);
+            }
+            for (PageId entry : c.getKeysetSortedByKey()) {
+                long count = c.getCount(entry);
+                if (count > minError) {
+                    errln(
+                            locale
+                                    + "\t"
+                                    + entry.getSectionId()
+                                    + "\t"
+                                    + entry
+                                    + "\thas too many entries:\t"
+                                    + count);
```

Comment:
- String.format will avoid taking up 6 lines
- include `minError` also so it's clear what the target value is